### PR TITLE
Add support for C++

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: Configurations/unix-Makefile.tmpl [hunk 1]
```diff
@@ -944,16 +949,26 @@ EOF
               $incs .= " -I".$withargs{zlib_include};
           }
       }
-      my $ecflags = { lib => '$(LIB_CFLAGS)',
-                      dso => '$(DSO_CFLAGS)',
-                      bin => '$(BIN_CFLAGS)' } -> {$args{intent}};
+      my $cc = '$(CC)';
+      my $cflags = '$(CFLAGS)';
+      if (grep /\.cc$/, @srcs) {
```
Comment (richsalz) [inline Configurations/unix-Makefile.tmpl]: this shoule be .cc|.cpp

**Review by richsalz — CHANGES_REQUESTED** [review]
One nit, add .cpp support, and this is approved.

**Comment by levitte** [comment]
> One nit, add .cpp support, and this is approved.

Done

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
Thank you.  Merged into master.

**Pull request closed without merging**